Publish on npm

I'd love to install this package via npm, and it's currently possible to do so using [GitHub URLs](https://docs.npmjs.com/files/package.json#github-urls) syntax in `package.json` - but it would be awesome if the package was actually published to npm.

There is already a [bootstrap-dialog](https://www.npmjs.com/package/bootstrap-dialog) package published by someone else, which seems to reference this repo, but is actually currently at 1.34.6, so it hasn't been updated in a while.

The [bootstrap3-dialog](https://www.npmjs.com/package/bootstrap3-dialog) name isn't taken, and since that's the name in this repo's `package.json`, perhaps you could claim that one?

For me, the advantage of using npm instead of Bower is that Bower doesn't have a built-in way of locking dependencies, which has caused me actual problems on multiple occasions, and secondly, I just don't want to use so many package managers - I already need Composer for PHP, Bundler for Ruby, and npm for JS, and it's getting pretty crowded.

Hi, any updates on this? I'm having a lot of trouble when shrinkwrapping (locking down the version, if you're not familiar with npm) the library installed directly from GitHub, and it would be really great if I could do it properly through npm.

The errors are likely some npm bug, but it only happens with this library, and it's the only one installed directly from GitHub:

```
npm ERR! Problems were encountered
npm ERR! Please correct and try again.
npm ERR! invalid: have bootstrap3-dialog@1.35.1 (expected: github:nakupanda/bootstrap3-dialog#v1.35.1) /home/levacic/project/node_modules/bootstrap3-dialog
npm ERR! extraneous: bootstrap3-dialog@1.35.1 /home/levacic/project/node_modules/bootstrap3-dialog
```

So I'm pretty certain that this isn't working quite as expected.

Thanks in advance, if I can help in any way, let me know, I'd be glad to do it.

--- a/README.md
+++ b/README.md
@@ -48,6 +48,14 @@ bower install bootstrap3-dialog
 
 ================
 
+## Install using npm
+
+```
+npm install --save bootstrap3-dialog
+```
+
+================
+
 ## Build instructions:
 
 Prepare:

